lib/web_ui/dev/test_platform.dart

```diff
@@ -213,8 +222,14 @@ To automatically create this file call matchGoldenFile('$filename', write: true)
 
     if (write) {
       // Don't even bother with the comparison, just write and return
+      print('Updating screenshot golden: $file');
       file.writeAsBytesSync(encodePng(screenshot), flush: true);
-      return 'Golden file $filename was updated. You can remove "write: true" in the call to matchGoldenFile.';
+      if (isUpdateScreenshotGoldens) {
+        // Do not fail tests when bulk-updating screenshot goldens.
+        return 'OK';
+      } else {
+        return 'Golden file $filename was updated. You can remove "write: true" in the call to matchGoldenFile.';
```

Comment:
Why keep the boolean write parameter in the matchGoldenFile function though?

Reply:
That's in case you want to update one individual screenshot. Even if you use `-t one_test.dart` it may contain multiple screenshot tests inside.

Reply:
Ahh yes, we can't target single tests, only files :'(